test/e2e/windows/service.go

```diff
@@ -61,11 +61,14 @@ var _ = SIGDescribe("Services", func() {
 
 		//using hybrid_network methods
 		ginkgo.By("creating Windows testing Pod")
-		windowsPod := createTestPod(f, windowsBusyBoximage, windowsOS)
-		windowsPod = f.PodClient().CreateSync(windowsPod)
+		testPod := createTestPod(f, windowsBusyBoximage, windowsOS)
```

Comment:
CreateTestPod sets the OS on the node selector: 

https://github.com/kubernetes/kubernetes/blob/3552bb5d8a893a8168cfa23001342aaebcd311d1/test/e2e/windows/hybrid_network.go#L103-L124
Do we know why it's not being respected?

Reply:
good catch ! ok ya we discussed that yesterday :) ... ok let me add a ginkgo.By to this PR, then do more testing, then revisit this in a few

Reply:
wait ! its in `jig.Run` changing this nowwww

Reply:
ive backed this out into a ginkgo.By, rather then a redundant os.windows step